server.js

```diff
@@ -67,16 +68,7 @@ app.use('/service-worker.js', serve('./dist/service-worker.js'))
 
 // 1-second microcache.
 // https://www.nginx.com/blog/benefits-of-microcaching-nginx/
-const microCache = LRU({
-  max: 100,
-  maxAge: 1000
-})
-
-// since this app has no user-specific content, every page is micro-cacheable.
```

Comment:
Can we preserve this block of comment? It's still necessary to remind users that you cannot microcache in every case.

Reply:
added back 👍